src/Execution/ValuesResolver.php

```diff
@@ -26,13 +27,22 @@
 use Digia\GraphQL\Util\ConversionException;
 use Digia\GraphQL\Util\TypeASTConverter;
 use Digia\GraphQL\Util\ValueASTConverter;
+use Exception;
+use Traversable;
+use function array_key_exists;
+use function array_keys;
+use function array_map;
+use function array_merge;
+use function Digia\GraphQL\Test\jsonEncode;
 use function Digia\GraphQL\Util\find;
+use function Digia\GraphQL\Util\invariant;
 use function Digia\GraphQL\Util\keyMap;
 use function Digia\GraphQL\Util\suggestionList;
+use function implode;
```

Comment:
I would just use \ for these core methods

Reply:
My bad. I let PhpStorm resolve ”issues” on it’s own. 😅

Reply:
There's an optin to ignore the root namespace

Reply:
I fixed this and updated the project settings.